pandas/tests/plotting/frame/test_frame.py

```diff
@@ -148,6 +148,23 @@ def test_plot(self):
         result = ax.axes
         assert result is axes[0]
 
+    def test_nullable_int_plot(self):
+        # GH 32073
+        dates = ["2008", "2009", None, "2011", "2012"]
+        df = pd.DataFrame(
+            {
+                "A": [1, 2, 3, 4, 5],
+                "B": [7, 5, np.nan, 3, 2],
```

Comment:
What about having a nullable value in column "A" (x-axis)?

Reply:
@ivanovmg I added the changes to the latest commit. Can you please review it again ?